src/platform/linux/misc.cpp

```diff
@@ -241,22 +241,24 @@ namespace platf {
 
   bp::child
   run_command(bool elevated, bool interactive, const std::string &cmd, boost::filesystem::path &working_dir, const bp::environment &env, FILE *file, std::error_code &ec, bp::group *group) {
+    // clang-format off
```

Comment:
clang-format gets confused by the `boost::process` syntax and thinks `bp::std_in` is a template class `bp::std_in <bp::null, bp::std_out>`